### PR TITLE
feat(chore): configure pre-commit hook

### PR DESCRIPTION
**Pull request by jamesgeorge007**
Now `lint` script is executed before making a commit on staged files.
Follow up of #84

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,6 +31,14 @@
...
     "release": "standard-version"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.js": "npm run lint"
+  },
```
Comment (anikethsaha) [inline package.json]: You had these in earlier PR too right ?

Comment (jamesgeorge007) [reply package.json]: Nope

Comment (jamesgeorge007) [reply package.json]: Ig you are talking about a similar PR to the `docsify` repo :thinking:

Comment (anikethsaha) [reply package.json]: ohh ok, my brain might be messing with me

**Pull request merged**